dev/NumberBox/NumberBox.cpp

```diff
@@ -195,9 +193,12 @@ void NumberBox::ValidateInput()
         {
             // Setting NumberFormatter to something that isn't an INumberParser will throw an exception, so this should be safe
```

Comment:
Is this true for setting NumberFormatter to nullptr as well?

Reply:
Yes, it will throw in ValidateNumberFormatter() if you try to set it to null.